lib/pharos/up_command.rb

```diff
@@ -112,8 +112,8 @@ def configure(config)
       craft_time = Time.now - start_time
       puts pastel.green("==> Cluster has been crafted! (took #{humanize_duration(craft_time.to_i)})")
       puts "    To configure kubectl for connecting to the cluster, use:"
-      puts "      #{$PROGRAM_NAME} kubeconfig > $HOME/.pharos/config"
-      puts "      export KUBECONFIG=$KUBECONFIG:$HOME/.pharos/config"
+      puts "      #{$PROGRAM_NAME} kubeconfig > kubeconfig"
+      puts "      export KUBECONFIG=./kubeconfig"
```

Comment:
That will only work as long as the user is in that directory and after that may cause connection to some other cluster. Maybe `$PWD/kubeconfig` to both?

Reply:
That said, it's also missing `-c` and `--tf-json` options.. idk how to fetch them from clamp (only if they are defined).

Reply:
`ARGV[1..-1]` seems to work

Reply:
Fixed.